src/Installer/redist-installer/targets/GenerateMSIs.targets

```diff
@@ -368,7 +371,7 @@
       InstallerFile="$(CombinedFrameworkSdkHostMSIInstallerFile)"
       WixExtensions="WixBalExtension;WixUtilExtension;WixTagExtension"
       WixSrcFiles="@(BundleMsiWixSrcFiles)"
-      AdditionalBasePaths="$(SdkPkgSourcesRootDirectory)">
+      AdditionalBasePaths="$(SdkPkgSourcesRootDirectory);$(PkgMicrosoft_DotNet_Build_Tasks_Installers)\build\wix\bundle">
```

Comment:
is this to ensure we don't find an extra wix on the path?

Reply:
That's just to keep the linker (Light.exe) happy because things are spread around a bit.